docs/guides/math.server.mdx

```diff
@@ -0,0 +1,70 @@
+export const navSortSelf = 3
+
+# Math
+
+This guide explores how to support math in markdown and MDX so that LaTeX can be
+included in content.
+MDX supports standard markdown syntax ([CommonMark][]).
+That means math is not supported by default.
+Math can be enabled by using [`remark-math`][remark-math] and either
+[`rehype-katex`][rehype-katex] (KaTeX) or [`rehype-mathjax`][rehype-mathjax]
+(MathJax).
+remark plugins can be passed in [`options.remarkPlugins`](#) and rehype plugins
+in [`options.rehypePlugins`](#).
+
+Say we have an MDX file like this:
```

Comment:
is may be worth noting these do not include the stylesheets.
The MathJax or Katex CSS needs to be included somewhere in the application for math to render correctly.

Reply:
It’s mentioned after the example (L43)